src/_guides/language/language-tour.md

```diff
@@ -1127,20 +1127,24 @@ is sometimes referred to as _arrow_ syntax.
   there, but you can use a [conditional expression](#conditional-expressions).
 {{site.alert.end}}
 
-A function can have two types of parameters: _required_ and _optional_.
-The required parameters are listed first, followed by any optional parameters.
-Optional parameters can be _named_ or _positional_.
+### Parameters
+
+A function can have any number of *required positional* parameters. These may be
+followed either by *optional positional* parameters or by *optional named*
+parameters.  A function can have optional positional parameters or optional
+named parameters, but not both.
```

Comment:
I think this can be a little shorter (and also we [avoid may](https://developers.google.com/style/word-list#letter-m)). Also, we should switch the order to match the section names... and maybe it isn't necessary to say "optional" before named — especially since we don't use "Optional" in the section name? 

```suggestion
A function can have any number of *required positional* parameters. These can be
followed either by *named* parameters or by *optional positional* parameters
(but not both).
```

Reply:
Done.